plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -0,0 +1,156 @@
+package io.jenkins.plugins.casc;
+
+import java.util.*;
```

Comment:
Your using IntelliJ right?

I wonder how you have configured your project, it seems to ignore the [`.idea/codeStyles`](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.idea/codeStyles/Project.xml)